feature-matcher-generator-e2e-test/src/test/java/ru/yandex/qatools/proc/test/MobTest.java

```diff
@@ -0,0 +1,61 @@
+package ru.yandex.qatools.proc.test;
+
+import org.hamcrest.Matcher;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Vladislav Bauer
+ */
+
+public class MobTest {
+
+    @Test
+    public void shouldFindMatcherForByteTypes() throws Exception {
+        checkMatcher("withValByte");
+        checkMatcher("withValueByte");
+    }
+
+    @Test
+    public void shouldFindMatcherForCharacterTypes() throws Exception {
+        checkMatcher("withValChar");
+        checkMatcher("withValueCharacter");
+    }
+
+    @Test
+    public void shouldFindMatcherForIntegerTypes() throws Exception {
+        checkMatcher("withValInt");
+        checkMatcher("withValueInteger");
+    }
+
+    @Test
+    public void shouldFindMatcherForLongTypes() throws Exception {
+        checkMatcher("withValLong");
+        checkMatcher("withValueLong");
+    }
+
+    @Test
+    public void shouldFindMatcherForFloatTypes() throws Exception {
+        checkMatcher("withValFloat");
+        checkMatcher("withValueFloat");
+    }
+
+    @Test
+    public void shouldFindMatcherForDoubleTypes() throws Exception {
+        checkMatcher("withValDouble");
+        checkMatcher("withValueDouble");
+    }
+
+    @Test
+    public void shouldFindMatcherForBooleanTypes() throws Exception {
+        checkMatcher("withValBoolean");
+        checkMatcher("withValueBoolean");
+    }
+
+    private void checkMatcher(String withValByte) throws NoSuchMethodException {
```

Comment:
please don't use the word "Check" because it means nothing in context of test. The fact of test already = checking smth. It should be action which we expect. For example here it should be `shouldSeeMatcher`